traffic_portal/app/src/common/modules/table/deliveryServices/TableDeliveryServicesController.js

```diff
@@ -235,10 +298,24 @@ var TableDeliveryServicesController = function(deliveryServices, $anchorScroll,
         null, // Divider
         {
             text: 'View Charts',
+            displayed: function () {
+                // only show if custom ds charts link is NOT configured
+                return !showCustomCharts;
+            },
             click: function ($itemScope) {
                 locationUtils.navigateToPath('/delivery-services/' + $itemScope.ds.id + '/charts?type=' + $itemScope.ds.type);
             }
         },
+        {
+            text: 'View Charts',
+            displayed: function () {
+                // only show if custom ds charts link IS configured
+                return showCustomCharts;
+            },
+            click: function ($itemScope, event) {
+                deliveryServiceUtils.openCharts($itemScope.ds, event);
```

Comment:
This is shadowing [the global `window.event`](https://developer.mozilla.org/en-US/docs/Web/API/Window/event) variable, I'd recommend not doing that to avoid confusion.